packages/db/postgres/tables.sql

```diff
@@ -132,8 +132,7 @@ CREATE INDEX IF NOT EXISTS pin_content_cid_idx ON pin (content_cid);
 CREATE INDEX IF NOT EXISTS pin_location_id_idx ON pin (pin_location_id);
```

Comment:
Don't forget to kill the (contained in `UNIQUE` )
```
CREATE INDEX IF NOT EXISTS pin_content_cid_idx ON pin (content_cid);
```

Reply:
yeah, just commited that change